### PR TITLE
[Docs] Improve docs

### PR DESCRIPTION
**Pull request by pilino1234**
Minor spelling/grammar/content fixes
Fix copy-paste errors
Formatting

### CODE REVIEW

**Comment by pilino1234** [comment]
WIP, found a couple of things I need to fix

- [ ] Using rst line blocks in tables to force line breaks
- [ ] Alecto WS1700 page is there twice

**Comment by CurlyMoo** [comment]
> Using rst line blocks in tables to force line breaks

One of the mistakes i indeed saw.

**Comment by pilino1234** [comment]
In the current tree, there are quite a lot of the forced line breaks, sometimes in places where they might not be necessary. However, not using them at all leads to scrollbars in the HTML view because the lines are too long (and are not broken automatically), which looks worse than having the forced linebreaks (see screenshot). I'll start putting them back where necessary/possible

screenshot_20171216_114507

**Comment by pilino1234** [comment]
This is pretty much done now. Please give it a (thorough) review 😄

I decided against using forced line breaks in tables because it's too much work to maintain for too little gain IMO.

Should I squash this be squashed into a single commit, or should it be split into a commit for the content fixes and a commit for the sphinx fixes?

File: docs/source/eventing/actions/dim.rst [hunk 1]
```diff
@@ -26,16 +26,16 @@ Dims a device to a dimlevel or starts a sequence from one dimlevel to another.
...
+| AFTER    | |no|             | |no|                | Time to wait before setting the new state             |
 +----------+------------------+---------------------+-------------------------------------------------------+
-| IN       | |no|             | |no|                | | To be combined with FROM. In what time do we want   |
-|          |                  |                     | | to change the dimlevel FROM x TO y                  |
+| IN       | |no|             | |no|                | To be combined with FROM. Duration of the dimming     |
+|          |                  |                     | process to change the dimlevel FROM x TO y            |
 +----------+------------------+---------------------+-------------------------------------------------------+
-| FROM     | |no|             | |no|                | | To be combined with IN. From what dimlevel do we    |
-|          |                  |                     | | want to start a sequence to the target dimlevel     |
+| FROM     | |no|             | |no|                | To be combined with IN. The dimlevel the progression  |
```
Comment (CurlyMoo) [inline docs/source/eventing/actions/dim.rst]: This doesn't seem right

File: docs/source/eventing/syntax.rst [hunk 2]
```diff
@@ -185,18 +185,18 @@ As you can also see, the fields (*state* or *dimlevel*) we can use depends on th
...
+In this case, an action will be triggered as soon as an ``arctech_switch`` code is received with a specific state, id, and unitcode. The ``arctech_switch`` does not have to be configured as an explicit device for this rule to work.
 
 Functions
 ---------
 
-In some cases, standard operators limit us in writing our rules. For example, calculating with time is a hideous task considering that hours do not go above 24, minute and seconds do not go above 60, and there are no negative numbers. Other functionality like randomization are also not possible in the standard event operators. This more advanced functionality is added in the form of function. A simple example:
+In some cases, standard operators limit us in writing our rules. For example, calculating with time is a complex task considering that hours do not go above 24, minute and seconds do not go above 60, and there are no negative numbers. Other functionality like randomization are also not possible in the standard event operators. This more advanced functionality is added in the form of function. A simple example:
```
Comment (CurlyMoo) [inline docs/source/eventing/syntax.rst]: `in the form of functions`

File: docs/source/programs/receive.rst [hunk 3]
```diff
@@ -46,13 +46,10 @@ Mandatory arguments to long options are mandatory for short options too.
...
 |  Specify protocols that should not be printed in the
-|  output stream. This is useful when trying to find
-|  IDs of devices such as weather stations or remote
-|  controls, but certain protocols are spamming the
-|  output. For example, datetime protocol data is
+|  output stream. For example, datetime protocol data is
 |  transmitted every second, and is likely to displace
 |  useful information in the output stream. It can
-|  then be hidden with --filter=datetime
+|  then be hidden with ``--filter=datetime``
```
Comment (CurlyMoo) [inline docs/source/programs/receive.rst]: Are you sure this works when compiling man-pages from the rst?

Comment (CurlyMoo) [reply docs/source/programs/receive.rst]: Ok, it does, because i used it before.

**Comment by CurlyMoo** [comment]
I would love to see this split in various parts.
1. Protocol documentation (in bits).
1.1 433.92
1.2 API 
1.3 Generic
2. Event stuff
3. Configuration
4. The rest

It easier to review when each PR is logically grouped and doesn't contain more then 10 to 15 changes files.

Another option is to keep this PR, but you make multiple commit logically group, so we can review each section one at a time. When everything is review, we squash.

File: docs/source/protocols/433.92/weather/alecto_wsd17.rst [hunk 4]
```diff
@@ -36,53 +34,55 @@ Alecto WSD17
...
+             "group": [ "Outside" ],
+             "media": [ "all" ]
+           }
+         }
        }
-     }
-   }
+
+    +------------------+-----------------+
+    | **Option**       | **Value**       |
```
Comment (CurlyMoo) [inline docs/source/protocols/433.92/weather/alecto_wsd17.rst]: This doesn't look right.

File: docs/source/protocols/api/weather/wunderground.rst [hunk 5]
```diff
@@ -56,27 +56,27 @@ Weather Underground
...
-+------------------+-------------------------------------+
-| sun              | rise / set                          |
-+------------------+-------------------------------------+
-| update           | 0 - 1                               |
-+------------------+-------------------------------------+
++------------------+--------------------------------------+
+| **Option**       | **Value**                            |
++------------------+--------------------------------------+
+| api              | *valid weather underground api key*  |
++------------------+------------------------------- ------+
```
Comment (CurlyMoo) [inline docs/source/protocols/api/weather/wunderground.rst]: Bogus space.

**Comment by CurlyMoo** [comment]
I scanned everything in general and read some part more closely and it seems fine.

**Comment by pilino1234** [comment]
I'll work on rebasing and splitting this into separate commits soon. I think keeping them all in the same PR and squashing after review is the cleanest approach.

**Comment by CurlyMoo** [comment]
> I think keeping them all in the same PR and squashing after review is the cleanest approach.

Yes, that's good. But having them reviews in blocks of 10 is better i think.

**Comment by CurlyMoo** [comment]
Any ideas?

**Pull request closed without merging**